source/performance/.globalconfig

```diff
@@ -2,9 +2,3 @@ is_global = true
 
 # CA1062: Validate arguments of public methods
 dotnet_diagnostic.CA1062.severity = none
-
-# CA1309: Use ordinal string comparison
-dotnet_diagnostic.CA1309.severity = none
```

Comment:
Moved "global suppressions" to local suppressions in source via the [SuppressMessageAttribute](https://learn.microsoft.com/dotnet/api/system.diagnostics.codeanalysis.suppressmessageattribute) for warnings that appear in only a few locations.